src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -41,12 +41,12 @@
              error (style/create-server-error-message error)
              :else [comps/Spinner {:text "Loading details..."}])))
    :component-did-mount
-   (fn [{:keys [this]}]
-     (this :load-agora-method))
+   (fn [{:keys [this props state]}]
+     (react/call :load-agora-method this props state))
```

Comment:
Actually, it doesn't look like you're using state? I see below you mention there's some issue with pulling props, but we should try to fix that.